packages/itwin/tree-widget/src/components/trees/models-tree/ModelsTreeDefinition.ts

```diff
@@ -35,6 +37,9 @@ import type {
   InstanceKey,
 } from "@itwin/presentation-shared";
 import type { ModelsTreeIdsCache } from "./internal/ModelsTreeIdsCache";
+import type { ClassGroupingNodeKey } from "@itwin/presentation-hierarchies/lib/cjs/hierarchies/HierarchyNodeKey";
```

Comment:
should import from barrel

Reply:
Hmm, apparently it's not exported thru barrel. There seems no other way to get this type from `hierarchies`. For the time being, I can copy this type, but we should think about whether or not hierarchy node key types should be exported.

Reply:
Agree. Can you please add those exports in [the associated PR](https://github.com/iTwin/presentation/pull/688)?